packages/fx-core/src/plugins/resource/function/constants.ts

```diff
@@ -25,7 +25,7 @@ export class FunctionPluginInfo {
     FunctionConfigKey.storageAccountName,
     FunctionConfigKey.appServicePlanName,
     FunctionConfigKey.functionEndpoint,
-    FunctionConfigKey.functionAppId,
+    FunctionConfigKey.functionAppResourceId,
```

Comment:
seems the key is changed? do we need update migration logic?

Reply:
Fixed.